spring-cloud-task-batch/src/test/java/org/springframework/cloud/task/batch/partition/DeployerPartitionHandlerTests.java

```diff
@@ -60,6 +60,7 @@
 
 /**
  * @author Michael Minella
+ * @author Glenn Renfro
  */
 public class DeployerPartitionHandlerTests {
```

Comment:
Based on my earlier comment about giving the user the option to pass in the id via env vars of command line args, we'll need a test for the env var option as well.

Reply:
Done